pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -170,9 +186,9 @@ protected BrokerResponse handleRequest(long requestId, String query, SqlNodeAndO
           requestContext, httpHeaders);
       if (!brokerResponse.getExceptions().isEmpty()) {
         // a _green_ error (see handleRequestThrowing javadoc)
-        LOGGER.info("Request {} failed in a controlled manner: {}", requestId, brokerResponse.getExceptions());
```

Comment:
I feel we just need to include stats in this log

Reply:
We also need the stats when the query finish successfully